File: Herencia/ejercicio 1/Java/Moto.java
public class Moto extends Vehiculo{
    public int cilindrada;
    public String motor;

    public Moto(String marca, String modelo, int año, double precio, int cilindrada, String motor) {
        super(marca, modelo, año, precio);
        this.cilindrada = cilindrada;
        this.motor = motor;
    }

    public String mostrar() {
        return super.mostrar() + "cilindrada: " + cilindrada + "motor: " + motor;
    }
}

File: Herencia/ejercicio 1/Python/Moto.py
from Vehiculo import Vehiculo
class Moto(Vehiculo):
    def __init__(self, marca, modelo, año, precio, cilindrada, motor):
        super().__init__(marca, modelo, año, precio)
        self.cilindrada = cilindrada
        self.motor = motor
    def mostrar(self):
        return super().mostrar(), "Cilindrada: ", self.cilindrada, "cc", "Tipo Motor:", self.motor